src/dmtcpworker.cpp

```diff
@@ -276,6 +276,7 @@ dmtcp_initialize()
   static bool initialized = false;
 
   if (initialized) {
+    buf = new jalib::JBuffer(0); // To force linkage of jbuffer.cpp
```

Comment:
The `buf =` should be part of teh `else` condition here.

Also, I suggest adding comment:
// Technically, this is a memory leak, but `rv` is static and so it happens only once.